src/test/java/org/broadinstitute/hellbender/tools/picard/fasta/dek_Normalizefasta_IT.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.picard.fasta;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+import org.broadinstitute.hellbender.tools.picard.fastq.NormalizeFasta;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.annotations.Test;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Created by dkling on 9/2/15.
+ */
+
+    public final class dek_Normalizefasta_IT extends CommandLineProgramTest {
+
+        private static final File TEST_DATA_PATH = new File(getTestDataDir(), "picard/fasta/" );
+
+        public String getTestedClassName() {
+            return NormalizeFasta.class.getSimpleName();
+        }
+
+        @Test
+        public void testNormalize () throws IOException {
+
+            final File input = new File(TEST_DATA_PATH, "exampleFASTA.fasta");
+            final File expectedFile = new File(TEST_DATA_PATH, "Normalizedexp.fasta");
+            final File outfile = BaseTest.createTempFile("normalized", ".fasta");
+
+            final String[] args = {
+                    "--INPUT", input.getAbsolutePath(),
+                    "--OUTPUT", outfile.getAbsolutePath(),
+            };
+
+            runCommandLine(args);
+            IntegrationTestSpec.assertEqualTextFiles(outfile, expectedFile, "#");
+
+        }
+
```

Comment:
Got everything working.  Then retested LINE_LENGTH=NULL.  Threw the following error:


java.lang.IllegalArgumentException: Can not set int field org.broadinstitute.hellbender.tools.picard.fastq.NormalizeFasta.LINE_LENGTH to null value
	at sun.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(UnsafeFieldAccessorImpl.java:167)
	at sun.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(UnsafeFieldAccessorImpl.java:171)
	at sun.reflect.UnsafeIntegerFieldAccessorImpl.set(UnsafeIntegerFieldAccessorImpl.java:80)
	at java.lang.reflect.Field.set(Field.java:764)
	at org.broadinstitute.hellbender.cmdline.CommandLineParser$ArgumentDefinition.setFieldValue(CommandLineParser.java:838)
	at org.broadinstitute.hellbender.cmdline.CommandLineParser.setArgument(CommandLineParser.java:402)
	at org.broadinstitute.hellbender.cmdline.CommandLineParser.parseArguments(CommandLineParser.java:268)
	at org.broadinstitute.hellbender.cmdline.CommandLineProgram.parseArgs(CommandLineProgram.java:183)
	at org.broadinstitute.hellbender.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:106)
	at org.broadinstitute.hellbender.Main.instanceMain(Main.java:71)
	at org.broadinstitute.hellbender.Main.instanceMain(Main.java:78)
	at org.broadinstitute.hellbender.CommandLineProgramTest.runCommandLine(CommandLineProgramTest.java:75)
	at org.broadinstitute.hellbender.tools.picard.fastq.NormalizeFastaIntegrationTest.testNormalize(NormalizeFastaIntegrationTest.java:38)

Process finished with exit code 0